### DISCUSSION TITLE
Add Button to Collapse/Expand All Categories

### DISCUSSION THREAD

**Discussion by ttreker** [home-assistant/frontend#20738] [Configuration]
### The title of this feature request I believe speaks for itself.
(Maybe I should add on all lists that can be categorized?)

This is one of no doubt many future improvements to this excellent first-pass at organization that can continue to reduce the frustration of keeping track of what goes into an abstract automation project.

#### Some Tangential Reflections
To express how happy I am with the Categories and Labels functionality would be very difficult. One way to demonstrate this would be to point out that quite frankly I dreaded adding new functionality or tracking down issues with old functionality because of having to navigate a sea of disorganized scripts, helpers, scenes and automations where there were a set of ones of interest that all had to interact to achieve a desired end.

I would just avoid tasks because they always made my head spin trying to keep track.

As wonderful as Categories are I keep going crazy collapsing all the categories of no interest to look through the category of current interest.

Just to note, though, for the record I have gotten so much done since these organization features were introduced I can't tell you. I have been able to dump my previous system of having to prepend names with my own categories. I cannot begin to tell you how happy I am those days are over!

**Comment by Innominatapersona** [comment]
Maybe also an option to start with everything collapsed would be appreciated
